Umbraco-Cloud/Set-Up/Working-Locally/index.md

```diff
@@ -5,80 +5,216 @@ versionTo: 10.0.0
+
+- [Video Tutorial](#video-tutorial)
+- [Cloning an Umbraco Cloud Project](#cloning-an-umbraco-cloud-project)
+- [Running the Site Locally](#running-the-site-locally)
+- [Working with Visual Studio](#working-with-visual-studio)
+    - [Adding a Solution File to your Cloud Project](#adding-a-solution-file-to-your-cloud-project)
+    - [Adding Additional Projects to your Solution](#adding-additional-projects-to-your-solution)
+- [Renaming the Project Files and Folders](#renaming-the-project-files-and-folders)
 
 ## Video Tutorial
 
 <iframe width="800" height="450" src="https://www.youtube.com/embed/e3spd6Nqrf8?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Cloning the Umbraco Project
+## Cloning an Umbraco Cloud Project
+
+To clone an Umbraco cloud project, follow these steps:
+
+1. On the Umbraco Cloud portal, **Go to the project** you wish to clone.
+2. Click on the arrow next to the **Development** environment and select **Clone project**.
+
+   ![Clone project option](images/clone-project-option.png)
+3. **Copy** the clone URL to copy the development environment's git repository endpoint.
+
+    ![Copy the clone URL](images/copy-clone-url.png)
+4. Go to the location on your local machine where you wish to clone the project.
+5. Clone the repository using your favorite Git client. For example: [Fork](https://git-fork.com/), [SourceTree](https://www.sourcetreeapp.com/), [Git Extensions](https://gitextensions.github.io), [GitKraken](https://www.gitkraken.com/), etc. For this article, I'll use **Git Bash**.
+6. In the **Git Bash** terminal, type the following command:
+
+   ```cs
+   git clone <Git clone URL>
+   ```
+
+    where the `<Git clone URL>` is the URL you copied from the cloud development environment.
+    ![Git Bash Terminal](images/git-bash-terminal.png)
+
+7. Press **Enter**.
+
+8. Once the project has been cloned down, you will get a folder either with the cloud project name or the name you gave the project.
+
+    ![Cloned Project](images/cloned-project.png)
+
+9. Now, you have an exact copy of your Umbraco Cloud Development environment that you can run locally.
+
+## Running the Site Locally
+
+To run Umbraco locally, you will need to [install the latest .NET SDK](https://dotnet.microsoft.com/download) (if you do not have this already).
+
+With dotnet installed, run the following commands in a terminal application of your choice. You can also refer to the `Readme` file in the project folder.
+
+1. Navigate to the newly created project folder and then run the following commands:
+
+    ```cs
+    cd src/UmbracoProject
+    ```
+
+2. Build and run the new Umbraco .Net Core project:
+
+    ```cs
+    dotnet build
+    dotnet run
+    ```
+
+3. The terminal output will show the application starting up and will include localhost URLs which you can use to browse to your local Umbraco site.
+
+    ![Terminal Output](images/terminal-output.png)
```

Comment:
This is not really a step, but information.
Consider if it makes sense to have this as part of the list. Could it perhaps be a note instead?